src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+  <tr>
+    <th></th>
+    <th>Single-region replication (synchronous)</th>
+    <th>Multi-region replication (synchronous)</th>
+    <th>Logical data replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</td>
+    <td>Immediate mode 0.5 seconds<br>Transactional mode ~10s of seconds</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Add application cutover time</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+    <td>Region-local latency depending on design<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Automatic</td>
+    <td>Automatic</td>
+    <td>Semi-automatic</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x<br>Networking between availability zones</td>
+    <td>1x<br>Networking between datacenters</td>
+    <td>2x<br>Networking between availability zones or datacenters</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Minimum regions</b>
+    </td>
+    <td>1</td>
+    <td>3</td>
+    <td>2</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Fault tolerance</b>
+    </td>
+    <td>Zero RPO node, availability zone failures</td>
+    <td>Zero RPO node, availability zone failures, region failures</td>
+    <td>Zero RPO node, availability zone, region failures with loss up to RPO</td>
+  </tr>
+
+</table>
+
+For details on designing your cluster topology for high availability with replication, refer to the [Disaster Recovery Planning]({% link {{ page.version.version }}/disaster-recovery-planning.md %}#hardware-failure) page.
+
+## Disaster recovery
+
+- [**Backup and point-in-time restore**]({% link {{ page.version.version }}/take-backups-with-revision-history-and-restore-from-a-point-in-time.md %}): While point-in-time backup and restore allows you to roll back to a specific point in time. Multiple supported [cloud storage]({% link {{ page.version.version }}/use-cloud-storage.md %}) providers means that you can store backups in an alternative provider. [Incremental backups]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}) allow you to configure backup frequency for lower [RPO]({% link {{ page.version.version }}/disaster-recovery-overview.md %}).
```

Comment:
Maybe want to mention Managed Backups in Cloud and the configurability features for Frequency and Retention?